tensorflow/core/util/mkl_util.h

```diff
@@ -1117,6 +1117,24 @@ inline memory::dims MklDnnDimsInNCHW(const memory::dims& in_dims,
   return memory::dims({n, c, h, w});
 }
 
+/// Overloaded version of function TFShapeToMklDnnDimsInNCDHW above.
+/// Input parameters are self-explanatory.
+inline memory::dims MklDnnDimsInNCDHW(const memory::dims& in_dims,
+                                      TensorFormat format) {
+  // Validate format.
+  CHECK_NE(TFDataFormatToMklDnnDataFormat(format),
```

Comment:
Could you please change this to `DCHECK`?

Reply:
Never mind. I think I'll fix it internally.

Reply:
Thanks